pages/_app.tsx

```diff
@@ -67,37 +68,64 @@ function App({ Component, pageProps }: MyAppProps): JSX.Element | null {
     return null;
   }
 
+  const theme = extendTheme({
+    breakpoints: {
+      xs: '640px',
+      sm: '768px',
+      mobile: '920px',
+      md: '1024px',
+      lg: '1280px',
+      lgx: '1440px',
+    },
+    colors: {
+      neutral: {
+        10: '#fafafa',
+        20: '#f5f5f5',
+        30: '#f0f0f0',
+        40: '#d9d9d9',
+        50: '#bfbfbf',
+        60: '#8c8c8c',
+        70: '#595959',
+        80: '#434343',
+        85: '#2c2c2c',
+        90: '#201f1f',
+      },
+    },
```

Comment:
Just migrated the css variables needed for this PR, we can progressive migrate the others later.